v1/entitlement/entitlement.gunk

```diff
@@ -0,0 +1,583 @@
+//                 },
+//                 "401": openapiv2.Response{
+//                         Description: "Returned when the request does not contains the user's credentials.",
+//                 },
+//                 "403": openapiv2.Response{
+//                         Description: "Returned when the user does not have permission to access the resource.",
+//                 },
+//                 "404": openapiv2.Response{
+//                         Description: "Returned when the resource is not found.",
+//                 },
+//                 "500": openapiv2.Response{
+//                         Description: "Returned when an unexpected error occured on the server side.",
+//                 },
+//         },
+//         SecurityDefinitions: openapiv2.SecurityDefinitions{
+//                 Security: map[string]openapiv2.SecurityScheme{
+//                         "OAuth2": openapiv2.SecurityScheme{
+//                                 Type:             openapiv2.TYPE_OAUTH2,
+//                                 Flow:             openapiv2.FLOW_ACCESS_CODE,
+//                                 Description:      "Access to entitlement resources is restricted to authenticated users. The authentication process follows the Authorization Code flow.",
+//                                 AuthorizationURL: "/v1/auth",
+//                                 TokenURL:         "/v1/token",
+//                                 Scopes: openapiv2.Scopes{
+//                                         Scope: map[string]string{
+//                                                 "read":  "Grants read access",
+//                                                 "write": "Grants write access",
+//                                         },
+//                                 },
+//                         },
+//                 },
+//         },
+// }
+package entitlement
+
+import (
+	"time"
+
+	"github.com/gunk/opt/http"
+	"github.com/gunk/opt/openapiv2"
+)
+
+type AddEntitlementRequestForCurrentUserRequest struct {
+	BankID   string `pb:"1" json:"bank_id"`
+	RoleName string `pb:"2" json:"role_name"`
+}
+
+type EntitlementRequest struct {
+	EntitlementRequestID string    `pb:"1" json:"entitlement_request_id"`
+	User                 User      `pb:"2" json:"user"`
+	RoleName             string    `pb:"3" json:"role_name"`
+	BankID               string    `pb:"4" json:"bank_id"`
+	Created              time.Time `pb:"5" json:"created"`
+}
+
+type EntitlementRequests struct {
+	EntitlementRequests []EntitlementRequest `pb:"1" json:"entitlement_requests"`
+}
+
+type List struct {
+	EntitlementID string `pb:"1" json:"entitlement_id"`
+	RoleName      string `pb:"2" json:"role_name"`
+	BankID        string `pb:"3" json:"bank_id"`
+}
+
+type Entitlements struct {
+	List []List `pb:"1" json:"list"`
+}
+
+type User struct {
+	UserID       string       `pb:"1" json:"user_id"`
```

Comment:
nit: use ID instead

Reply:
well, some of the existing ones use `user_id`.
![image](https://user-images.githubusercontent.com/7168293/68285672-b6851400-00a5-11ea-9b5c-a3c2ce6479c0.png)